buildspec-release.yml

```diff
@@ -12,14 +12,14 @@ phases:
       # run unit tests
       - AWS_ACCESS_KEY_ID= AWS_SECRET_ACCESS_KEY= AWS_SESSION_TOKEN=
         AWS_CONTAINER_CREDENTIALS_RELATIVE_URI= AWS_DEFAULT_REGION=
-        tox -e py27,py36 -- test/unit
+        tox -e py27,py37 -- test/unit
```

Comment:
should we do both `py36` and `py37`? I think there could be more of an argument against keeping both here (as opposed to the training toolkit), and that the answer depends on how tightly-coupled we want our framework toolkits to be with the DLC images.